app/src/ui/add-repository/create-repository.tsx

```diff
@@ -272,10 +293,52 @@ export class CreateRepository extends React.Component<ICreateRepositoryProps, IC
     )
   }
 
+  private renderInvalidPathWarning() {
+    const isValidPath = this.state.isValidPath
+    const pathSet = isValidPath !== null
+
+    if (!pathSet || isValidPath) {
+      return null
+    }
+
+    return (
+      <DialogError>
```

Comment:
Probably out of scope for the PR, but the red background when this is drawn stands out excessively when it's shown inline, rather than as the header. 

![](https://user-images.githubusercontent.com/1715082/27202529-9991d5aa-51e7-11e7-87b1-72b17d58b99d.png)

versus

<img width="423" alt="screen shot 2017-06-18 at 10 21 19 pm" src="https://user-images.githubusercontent.com/359239/27263922-94c64648-5474-11e7-8c48-95ddcf1369cf.png">

We can :boot: this to a follow-up PR as we don't have the styles for this currently anyway.